mlflow/server/js/src/components/HomeView.js

```diff
@@ -22,7 +22,7 @@ class HomeView extends Component {
   };
 
   state = {
-    listExperimentsExpanded: true,
+    listExperimentsExpanded: process.env.HIDE_EXPERIMENT_LIST !== 'true',
```

Comment:
Does this hide it for good or just leave it unexpanded? I think we just want to hide it. This will still show the button for opening it.